### PR TITLE
Add git SHA, tag and deployed date to files

### LINKED ISSUE #27: Add GIT_SHA, GIT_DEPLOYED, and GIT_TAG handling
We need this update to help ensure we know which version is currently deployed.
See other SOAR applications for functionality.

### PR DESCRIPTION
**Pull request by edthedev**
Closes #27

### CODE REVIEW

File: src/phsoar_null_router/soar_null_router_connector.py [hunk 1]
```diff
@@ -21,6 +21,9 @@
 import json
 from bs4 import BeautifulSoup
 
+__version__ = 'GITHUB_TAG'
+__git_hash__ = 'GITHUB_SHA'
+__deployed__ = 'GITHUB_DEPLOYED'
```
Comment (mpitcel) [inline src/phsoar_null_router/soar_null_router_connector.py:27]: The error seems to indicate that this requires a second blank line here

File: src/phsoar_null_router/soar_null_router_connector.py [hunk 2]
```diff
@@ -21,6 +21,9 @@
...
```
Comment (edthedev) [inline src/phsoar_null_router/soar_null_router_connector.py:26]:
```suggestion
__deployed__ = 'GITHUB_DEPLOYED'


```

**Review by mpitcel — APPROVED** [review]

File: src/phsoar_null_router/soar_null_router_connector.py (hunk 1 above)
Comment (edthedev) [reply src/phsoar_null_router/soar_null_router_connector.py:27]: Good call. Done.

**Review by zdc217 — APPROVED** [review]

File: src/phsoar_null_router/soar_null_router_connector.py (hunk 1 above)
Comment (ddriddle) [reply src/phsoar_null_router/soar_null_router_connector.py:27]:
```suggestion


```

Comment (edthedev) [inline src/phsoar_null_router/soar_null_router_connector.py:27]:
```suggestion
```

**Review by mpitcel — APPROVED** [review]

**Review by ddriddle — APPROVED** [review]

**Pull request merged**